extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/AbstractCache.java

```diff
@@ -11,7 +11,7 @@
 
     private Object defaultKey;
 
-    protected abstract String getName();
+    public abstract String getName();
```

Comment:
This is really necessary (the names collection could be sorted before retrieving the cache instances), but it doesn't hurt to have this method public so I'm fine with either.

Reply:
It's used in the view.

Reply:
Oh sorry, I missed that. Then it's perfectly fine :)